plugins/rest/plugin.spec.yaml

```diff
@@ -223,7 +223,7 @@ actions:
       body:
         title: Body
         description: Payload to submit to the server when making the HTTP Request call
-        type: object
+        type: '[]object'
         required: false
         example: '{"user": "user@example.com"}'
```

Comment:
Is this example correct according to the changes of input's `type`? 😃

Reply:
Oh no it's not, good catch thank you :D